src/pfe/portal/modules/projectInitializer.js

```diff
@@ -189,10 +189,17 @@ const createEnvForContainerCreation = (user, projectName, gitInfo) => {
   env.push(`CW_EXTENSIONS=${JSON.stringify(cwExtensions)}`);
 
   if (projectName && gitInfo.repo) {
-    env.push(
-      `PROJ_NAME=${projectName}`,
-      `GIT_REPO=${gitInfo.repo}`,
-    );
+    if (gitInfo.repo.endsWith("tar.gz")) {
```

Comment:
A minor nit: while you're making changes, could you make this check for `.tar.gz`?  (add dot in the front)